### PR TITLE
Kamila/fix fragment tests

### PR DESCRIPTION
**Pull request by kamilababayeva**
This pr:

- fixes layout in fragments
- fix the scoreboard tests

### CODE REVIEW

File: app/src/main/java/ch/sdp/vibester/fragment/ScoreBoardFragment.kt [hunk 1]
```diff
@@ -40,29 +40,24 @@ class ScoreBoardFragment : Fragment(), OnItemClickListener, View.OnClickListener
...
+        view.findViewById<Button>(R.id.scoreboard_topTracksButton).setOnClickListener{ setGenreListeners(view, "top tracks") }
+        view.findViewById<Button>(R.id.scoreboard_imagDragonsButton).setOnClickListener{ setGenreListeners(view, "Imagine Dragons") }
+        view.findViewById<Button>(R.id.scoreboard_billieEilishButton).setOnClickListener { setGenreListeners(view, "Billie Eillish") }
         setupRecycleView(view, ctx)
-
-        return view
     }
 
-    private fun selectScoreboard() {
+    private fun selectScoreboard(view:View) {
```
Comment (jiabaow) [inline app/src/main/java/ch/sdp/vibester/fragment/ScoreBoardFragment.kt:56]:
```suggestion
    private fun selectScoreboard(view: View) {
```

File: app/src/main/java/ch/sdp/vibester/fragment/ScoreBoardFragment.kt [hunk 2]
```diff
@@ -121,28 +116,8 @@ class ScoreBoardFragment : Fragment(), OnItemClickListener, View.OnClickListener
...
-                R.id.scoreboard_rockButton -> {
-                    genre = "rock";selectScoreboard()
-                }
-                R.id.scoreboard_topTracksButton -> {
-                    genre = "top tracks";selectScoreboard()
-                }
-
-            }
-
+    private fun setGenreListeners(view:View, genre:String){
```
Comment (jiabaow) [inline app/src/main/java/ch/sdp/vibester/fragment/ScoreBoardFragment.kt:119]:
```suggestion
    private fun setGenreListeners(view: View, genre: String){
```

**Review by MaximeZmt — APPROVED** [review]
😀 👍 
Thanks for the fix

**Review by jiabaow — APPROVED** [review]
👍 Happy to see they are back!

**Pull request merged**